src/robotlibcore.py

```diff
@@ -142,6 +135,38 @@ def get_keyword_documentation(self, name):
             doc = '{}\n\n{}'.format(doc, tags) if doc else tags
         return doc
 
+    def get_keyword_types(self, keyword_name):
+        method = self.keywords.get(keyword_name)
+        if keyword_name == '__init__':
+            method = self.__init__
+        elif keyword_name.startswith('__') and keyword_name.endswith('__'):
```

Comment:
1. Perhaps this should be done only `if not method`?
2. A comment saying that this prepares for RF asking types for special methods using `__xxx__` syntax.
3. The above code already handles `__init__` and I don't know any other special method we'd be interested to get types from. Perhaps this isn't needed at all in this context?

Reply:
Well, lets be prepared of there is need.